_utilities/layout-grid.md

```diff
@@ -504,116 +357,173 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
   </div>
 </div>
 
-<h2 id="sass-mixins">Sass mixins</h2>
-When generating your CSS from USWDS source files, you have the option of customizing many system defaults by modifying project theme variables. USWDS also provides grid mixins for adding grid functionality to custom semantic component CSS.
-
-### Variables
-Variables and maps determine the number of columns, gutter width, and media-query point at which to begin floating columns. We use variables to generate the predefined grid classes documented on this page, as well as for the custom mixins noted under [Spacing settings](#spacing-settings) and [Utilities settings](#utilities-settings).
-
-#### Spacing settings
-
-{% highlight scss %}
-// Values are set as units tokens.
-
-$theme-column-gap-sm:               2px;
-$theme-column-gap-md:               2;
-$theme-column-gap-lg:               3;
-$theme-column-gap-mobile:           2;
-$theme-column-gap-desktop:          4;
-$theme-grid-container-max-width:    'desktop';
-$theme-site-margins-breakpoint:     'desktop';
-$theme-site-margins-width:          4;
-$theme-site-margins-mobile-width:   2;
-{% endhighlight %}
-
-#### Utilities settings
-
-{% highlight scss %}
-// Turn on or off breakpoints
-$theme-utility-breakpoints: (
-  'card':              false,   // 160px
-  'card-lg':           false,   // 240px
-  'mobile':            false,   // 320px
-  'mobile-lg':         true,    // 480px
-  'tablet':            true,    // 640px
-  'tablet-lg':         false,   // 880px
-  'desktop':           true,    // 1024px
-  'desktop-lg':        false,   // 1200px
-  'widescreen':        false,   // 1400px
-);
-{% endhighlight %}
-
-### Mixins
-Mixins can be used in conjunction with grid variables to add grid functionality to semantic component Sass.
-
-{% highlight scss %}
-// Creates a wrapper for a series of rows
-// $container-size can be mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, or widescreen
-@include grid-container;
-@include grid-container($container-size);
-
-// Creates a wrapper for a series of columns
-@include grid-row;
+## Theme settings
+You can redefine column gap token sizes, set the grid container max width, and update site margin by defining the relevant theme setting in your `_uswds-theme.scss` file.
 
-// Specify the width between columns
-// $gap-size can be sm, md, lg, 2px, 05, 1, 3, 4, 6
-@include grid-gap;
-@include grid-gap($gap-size);
+Visit the [settings page]({{ site.baseurl }}/documentation/settings) for more information and available settings.
```

Comment:
**issue** We should point users directly to the list of grid settings (or at least as close as we can). Here is a possible suggestion to work with the suggested edit above:
```suggestion
Find a full list of the available [grid settings]({{ site.baseurl }}/documentation/settings#spacing-settings) on the settings page.
```

Reply:
I like this. Great suggestion! Resolved in [87c040b](https://github.com/uswds/uswds-site/pull/2276/commits/87c040b478aa2922ab341db331b65a95f68c7e1d)